docs/machine-learning/tutorials/sentiment-analysis-model-builder.md

```diff
@@ -60,63 +54,68 @@ Each row in the *wikipedia-detox-250-line-data.tsv* dataset represents a differe
 1 | == OK! ==  IM GOING TO VANDALIZE WILD ONES WIKI THEN!!!
 0 | I hope this helps.
 
+## Create a Model Builder config file
+
+When first adding a machine learning model to the solution it will prompt you to create an `mbconfig` file. The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
+
+1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
+1. In the dialog, name the Model Builder project **SentimentAnalysis**, and click **Add**.
+
 ## Choose a scenario
 
 ![Model Builder wizard in Visual Studio](../media/model-builder-scenarios.png)
 
 To train your model, you need to select from the list of available machine learning scenarios provided by Model Builder.
 
-1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
-1. For this sample, the scenario is sentiment analysis. In the *scenario* step of the Model Builder tool, select the **Sentiment Analysis** scenario.
+1. For this sample, the task is text classification. In the *Scenario* step of the Model Builder extension, select the **Text classification** scenario.
```

Comment:
I think if you do not have another item to say , you do not need `1`, 
when I saw I was expecting 2.... but it should not be 
![image](https://user-images.githubusercontent.com/36615490/200399688-fe1020fa-364f-41ab-9b97-e7c71b9c73dd.png)

Reply:
Resolved.